runtime/src/lib.rs

```diff
@@ -240,9 +240,22 @@ impl pallet_sudo::Config for Runtime {
 
 impl pallet_ethereum_chain_id::Config for Runtime {}
 
+pub const GAS_PER_SECOND: u128 = 4_000_000;
+pub const WEIGHT_PER_GAS: u128 = WEIGHT_PER_SECOND as u128 / GAS_PER_SECOND;
+pub struct GasMapper {}
```

Comment:
```suggestion
pub struct MoonbeamGasWeightMapping;
```

Reply:
My initial algorithm wasn't using a ratio (to be a bit more accurate) and so required to have a larger number to handle overflow.
I'm reverting to u64.

Reply:
I discussed with Wei and he will change the gas from usize (dynamic type) to u64. That will simplify the code too